apps/pythonapi/examples/camera_example.py

```diff
@@ -0,0 +1,33 @@
+# %% [md]
+#
+# # Controlling the Camera
```

Comment:
Done.

Reply:
Not seeing them in the current revision

Reply:
At the top of every cell with a camera command there is now a `help()` call for the relevant function. When you run the notebook it will print the documentation for the relevant function in that cell.